scripts/generate-coverage.sh

```diff
@@ -6,7 +6,7 @@
 set -e
 echo "" > coverage.txt
 go test -i -race ./cmd/odo
-for d in $(go list ./... | grep -v vendor | grep -v tests | grep -v testingutil); do
```

Comment:
Isn't this going to do quite the opposite what title of this PR says?
`grep -v` does invert match. So it actually excludes `testingutil`

Reply:
My plan is not to exclude ```testingutil```. I just remove the last grep so that pkg ```testingutil``` will be appear in the go list.

<img width="1104" alt="Screen Shot 2020-01-31 at 6 50 20 PM" src="https://user-images.githubusercontent.com/19851115/73542594-0d0eae00-445b-11ea-8830-ca4dcf589b0c.png">

Reply:
Oh, sorry. My bad. I was reading the diff in the other way (like you added the `grep -v testingutil`).
Not I see that you are removing it 😇 

I'll blame it on Friday :-D 
Sorry